ledger-core/Makefile

```diff
@@ -77,14 +77,15 @@ generate-proto-options:
 .PHONY: generate-protobuf
 generate-protobuf: protoc-gen-ins ## generate protobuf structs
 	protoc -I./vendor -I./ --ins_out=./ --proto_path=${GOPATH}/src insolar/payload/payload.proto
-	protoc -I./vendor -I./ --ins_out=./ --proto_path=${GOPATH}/src insolar/payload/proto_vnvn.r0.proto
 	protoc -I./vendor -I./ --gogoslick_out=./ insolar/node/node.proto
 	protoc -I./vendor -I./ --gogoslick_out=./ insolar/pulsestor/pulse.proto
 	protoc -I./vendor -I./ --gogoslick_out=./ --proto_path=${GOPATH}/src network/hostnetwork/packet/packet.proto
 	protoc -I./vendor -I./ --gogoslick_out=./ --proto_path=${GOPATH}/src network/consensus/adapters/candidate/profile.proto
 	protoc -I./vendor -I./ --gogoslick_out=./ network/servicenetwork/watermillmessage.proto
 	protoc -I./vendor -I./ --gogoslick_out=./ instrumentation/instracer/span_data.proto
 
+	cd insolar/payload; protoc -I../../vendor -I./ --ins_out=./ --proto_path=${GOPATH}/src proto_vnvn.r0.proto
```

Comment:
well, it could be done done by just protoc -I./insolar/payload instead of cd

Reply:
Actually there were problems yesterday, @ita-sammann and @cyraxred were trying to overcome it yesterday, i do not know status of that. You can pm me for details

Reply:
simplified this case